articles/gitops-mode-software.md

```diff
@@ -0,0 +1,33 @@
+# Manage software using GitOps
+
+This guide is a walkthrough on how to manage software using GitOps. In Fleet, there are three types of software: [Fleet-maintained apps](#fleet-maintained-apps), [App Store (VPP) apps](#app-store-vpp-apps), and [custom packages](#custom-packages).
+
+When using Gitops to manage Fleet, you can optionally put the UI in [GitOps mode](https://fleetdm.com/guides/gitops-mode). This prevents you from making changes in the UI that would be overridden by GitOps workflows.
+
+## Fleet-maintained apps
+
+Currently, managing [Fleet-maintained apps](https://fleetdm.com/guides/fleet-maintained-apps) is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (Gitops support coming soon. Currently, Gitops won't delete/modify Fleet-maintained apps configured in the UI).
+
+## App Store (VPP) apps
+
+To configure App Store (VPP) apps, via GitOps, please see the [Software section](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#software) of Fleet's best practice [GitOps documentation](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#gitops). Note that VPP apps must first be added to Apple Business Manager.
+
+## Custom packages
+
+To use the Fleet UI to manage custom packages, first turn GitOps mode on in **Settings > Integration > Change management**
```

Comment:
```suggestion
To configure custom packages via GitOps, please see the [`packages`](https://fleetdm.com/docs/configuration/yaml-files#packages) of Fleet's best practice [GitOps documentation](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#gitops).

If you want to use Fleet to host custom packages instead of a third-party package hosting tool (ex. [Artifactory](https://jfrog.com/artifactory/)), first turn GitOps mode on in **Settings > Integration > Change management**
```

I agree with Tim that we want to include the why here. Maybe something like the above.